src/dune_formatter.ml

```diff
@@ -23,7 +23,7 @@ let get_formatter toolchain ~document ~options:_ ~token:_ =
     match output with
     | Ok newText -> Some [ TextEdit.replace ~range ~newText ]
     | Error msg ->
-      message `Error "Dune formatting failed: %s" msg;
+      show_message `Error "Dune formatting failed: %s" msg;
```

Comment:
Please revert this. No additional clarity is gained here.

Reply:
I can for sure revert this but would love to keep it because at the time `message` was confusing to me and may be confusing to others because it doesn't say what kind of message that is:

- lsp and extension use messages to communicate; is that the `message`? 
- does `message` simply log somewhere?

Having either `popup_message` or `show_message` hints much more about the fact that it's being displayed.

(don't like taking your time to argue about naming; will revert if you think that's better)

Reply:
> does message simply log somewhere?

If it logged, it would be called "log".

The purpose of this alias was to avoid the annoyingly verbose `show{Information,Error,Debug}Message` functions. If our alias is just as verbose, then I don't see the point of it.

Indeed, I don't have much time to argue about this. Feel free to keep whatever name you like